src/common/gui/DisplayInfo.h

```diff
@@ -0,0 +1,22 @@
+/*
+** Information about the physical display upon which I reside
+**
+** As with UserInteractions, this is just a header; the actual implementations
+** are in src/(mac|win|linux)/(Mac|Win|Linux)DisplayInformation.cpp
+**
+** Most of these APIs will require a reference to a UI object of some form
+** so make each of the functions take a CFrame *
+*/
+
+#pragma once
+
+#include "vstcontrols.h"
+
+namespace Surge
+{
+namespace GUI
+{
+    float  getDisplayBackingScaleFactor(CFrame *); // this is in units of 1.0 == 1.0
+    CRect  getScreenDimensions(CFrame *);
```

Comment:
See my comment below about namespaces and indentation. I don't think this a good idea to indent these. I don't understand the comment at all and would put it on top (1.0 == 1.0??).

Reply:
For your edification, here's the comment I replaced it with

```
/*
** Return the backing scale factor. This is the scale factor which maps a phyiscal
** pixel to a logical pixel. It is in units that a high density display (so 4 physical
** pixels in a single pixel square) would have a backing scale factor of 2.0.
**
** We retain this value as a float and do not scale it by 100, like we do with
** user specified scales, to better match the OS API
*/    
```

which is way better